src/Dialogs/Airspace/AirspaceList.cpp

```diff
@@ -167,7 +167,35 @@ static constexpr StaticEnumChoice type_filter_list[] = {
   { CLASSG, _T("Class G") },
   { MATZ, _T("MATZ") },
   { RMZ, _T("RMZ") },
-  nullptr
+  //
```

Comment:
Can you remove this empty comment please?

Reply:
(or give it meaning)

Reply:
Empty comment removed.